system-tests/projects/experimentalSingleTabRunMode/webpack.config.js

```diff
@@ -1 +1,3 @@
-module.exports = {}
+module.exports = {
+  stats: 'errors-warnings',
```

Comment:
Webpack compilation will log the number of modules generated e.g. `30 modules`, but where this is logged is non-deterministic so I've disabled the logging here. I kept errors-warnings in case something happens in the future that we would like to see.